docs/source/common_issues.mdx

```diff
@@ -0,0 +1,77 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Common issues encountered when using PEFT
+
+If you encounter any issue when using PEFT, please check the following list of common issues and their solutions.
+
+## I used or modified an existing example but it does not work
+
+Please ensure that the versions of packages you use are up-to-date. Examples often rely on the most recent versions being used. In particular, check the version of the following packages:
+
+- `peft`
+- `transformers`
+- `accelerate`
+- `torch`
+
+In general, you can update the version of your package by running this command inside your Python environment:
+
+```bash
+python -m pip install -U <package_name>
+```
+
+Especially for PEFT, it may be useful to install the package from source:
+
+```bash
+python -m pip install git+https://github.com/huggingface/peft
+```
+
+## I successfully trained a model but when I load it, I get bad results
+
+There can be several reasons for this. Please check one of the proposed solutions below.
+
+### Random deviations
+
+If your model outputs are not exactly the same as previously, there could be an issue with random elements. For example, when the model uses dropout, please ensure that it is in `.eval()` mode. If you use `.generate()` on a language model, there could be random sampling, so obtaining the same result requires a random seed to be set. Finally, if you used quantization and merged the weights, small deviations are expected due to rounding errors.
+
+### Correctly loading the model
```

Comment:
Since we're discussing what's wrong, the headers should describe the issue rather than the solution

```suggestion
### Incorrectly loaded model
```